packages/core/src/colors/index.ts

```diff
@@ -9,4 +14,22 @@ export const defaultTheme = {
   shadow: '#0a0a0a'
 } satisfies LikeC4Theme
 
+export function computeColorValues(color: HexColorLiteral): ThemeColorValues {
+  const colors = generate(color)
+
+  return {
+    elements: {
+      fill: color,
+      stroke: colors[6] as HexColorLiteral,
+      hiContrast: colors[1] as HexColorLiteral,
+      loContrast: colors[2] as HexColorLiteral
+    },
```

Comment:
If we use [@mantine/colors-generator](https://github.com/mantinedev/mantine/blob/master/packages/%40mantine/colors-generator/src/generate-colors.ts#L44C17-L44C31)

```suggestion
    elements: {
      fill: color[6] as HexColorLiteral,
      stroke: colors[7] as HexColorLiteral,
      hiContrast: colors[0] as HexColorLiteral,
      loContrast: colors[1] as HexColorLiteral
    },
```